### PR TITLE
Update tank_utility.markdown

### PR DESCRIPTION
**Pull request by neillt**
Provided clarity on requirement for double quotes on each device ID in the configuration.yaml file.  Also corrected configuration markdown to property denote the devices stanza, and not "unit_of_measurement" which is not part of this integration.  Fixed example YAML to include double quotes for those copying and pasting.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #73296

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @neillt, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#23081).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/tank_utility.markdown [hunk 1]
```diff
@@ -75,8 +75,8 @@ password:
   description: "Your [https://app.tankutility.com](https://app.tankutility.com) password."
   required: true
   type: string
-unit_of_measurement:
-  description: All devices to monitor.
+devices:
+  description: All devices to monitor.  Double quotes have been reported as required around each device string to configure the integration.
```
Comment (frenck) [inline source/_integrations/tank_utility.markdown]: This is not a device/integration issue. This is about YAML. I don't think we should describe how YAML works hidden in a note at a configuration option.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tank_utility.markdown (hunk 1 above)
Comment (neillt) [reply source/_integrations/tank_utility.markdown]: OK fair enough.  Will edit out that sentence, other changes remain.

**Comment by neillt** [comment]
@frenck changes made as requested.

**Review by frenck — APPROVED** [review]
Thanks, @neillt 👍

**Pull request merged**